package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "slimerjs-core",
+  "version": "1.0.0",
```

Comment:
I think we should probably follow sem-var semantics (breaking versions for major version nubmers), but happy to continue the existing slimerjs package versioning updates for now. Will update this.